svf-llvm/include/SVF-LLVM/ObjTypeInference.h

```diff
@@ -105,6 +109,14 @@ class ObjTypeInference
 
     u32_t getArgPosInCall(const CallBase *callBase, const Value *arg);
 
+public:
+    /// get or infer the name(s) of thisptr
+    Set<std::string> &inferThisPtrClassName(const Value *thisPtr);
+
+protected:
+
+    /// Backward collect all possible cpp constructors starting from a value
+    Set<const Value *> findCPPSources(const Value *startValue);
```

Comment:
FindCPPConstor?

Reply:
sorry for the confusion, sources can be heap allocation or self-inference functions (constructors/destructors or template functions). I've updated the comments.